### DISCUSSION TITLE
python virtual env does not shown in terminal

### DISCUSSION THREAD

**Discussion by ankur-gupta-29** [ohmybash/oh-my-bash#456] [Q&A]
OMB_PROMPT_SHOW_PYTHON_VENV=true  # enable
i want to show virtual env of python but that do not working

**Comment by akinomyoga** [comment]
It is only shown when the theme specified by `OSH_THEME` supports it. The feature request to support it in a theme is accepted. Which theme do you use? (The theme is the one specified by `OSH_THEME=...` in your `~/.bashrc`.)

**Reply by ankur-gupta-29** [reply]
kitsune, if this theme does not support then how I change this theme so it will support this?

**Reply by akinomyoga** [reply]
> how I change this theme so it will support this?

Thank you for your reply. First, you need to read "*[Customization of Plugins and Themes](https://github.com/ohmybash/oh-my-bash#customization-of--plugins-and-themes)*" in README. Then, you can add the following piece of code in the function `_omb_theme_PROMPT_COMMAND` defined in the `kitsune` theme file somewhere before setting `PS1`.

```bash
  local python_venv
  _omb_prompt_get_python_venv
```

Then, you can reference the variable `$python_venv` in variable `PS1`. The styles may be adjusted using the variables `OMB_PROMPT_VIRTUALENV_FORMAT` and `OMB_PROMPT_CONDAENV_FORMAT`. These variables can be, for example, configured in the following way:

```bash
OMB_PROMPT_VIRTUALENV_FORMAT="[venv %s]"
OMB_PROMPT_CONDAENV_FORMAT="[conda %s]"
```

where `%s` would be replaced with the actual value. The variables can contain color codes.

----

If you can specify the position and the styles of the python venv information in the prompt, I can implement it and add it to the kitsune theme.

**Reply by ankur-gupta-29** [reply]
┌─[ankur@ankur-desktop]─(~)
└─[12:31]-(^_^)-[$] 
after the  () bracket near the user name and thank you for your response

**Reply by akinomyoga** [reply]
I have added the python-venv segment to `kitsune` in the latest push to the master branch. You need to turn on it by setting the variable `OMB_PROMPT_SHOW_PYTHON_VENV` in your `~/.bashrc` as described in [README](https://github.com/ohmybash/oh-my-bash#enabledisable-python-venv). Could you check its behavior?

**Reply by ankur-gupta-29** [reply]
this is not working so I have made some changes to that so now it is working.I am created a pull request.
